solutions/observability/logs/streams/management/extract.md

```diff
@@ -0,0 +1,155 @@
+---
+applies_to:
+    serverless: preview
+---
+# Extract fields [streams-extract-fields]
+
+Unstructured log messages need to be parsed into meaningful fields so you can filter and analyze them quickly. Common fields to extract include timestamp and the log level, but you can also extract information like IP addresses, usernames, or ports.
+
+Use the **Extract field** tab on the **Manage stream** page to process your data. Changes are immediately available as a preview and tested end-to-end.
+The UI simulates your changes, so you can see them immediately.
+
+The UI also shows indexing problems, such as mapping conflicts, so you can address them before applying changes.
+
+:::{note}
+Applied changes aren't retroactive and only affect *future data ingested*.
+:::
+
+## Add a processor [streams-add-processors]
+
+Streams uses {{es}} ingest pipelines to process your data. Ingest pipelines are made up of processors that transform your data.
+
+To add a processor:
+
+1. Select **Add processor** to open a list of supported processors.
+1. Select a processor from the list:
+    - [Date](./extract/date.md)
+    - [Dissect](./extract/dissect.md)
+    - [Grok](./extract/grok.md)
+    - [Key-Value (KV)](./extract/key-value.md)
+    - GeoIP
+    - Rename
+    - Set
+    - URL Decode
+1. Select **Add Processor** to save the processor.
+
+:::{note}
+Editing processors with JSON is planned for a future release. More processors may be added over time.
+:::
+
+### Add conditions to processors [streams-add-processor-conditions]
+
+You can provide a condition for each processor under **Optional fields**. Conditions are boolean expressions that are evaluated for each document. Provide a field, a value, and a comparator.
+Processors support these comparators:
+- equals
+- not equals
+- less than
+- less than or equals
+- greater than
+- greater than or equals
+- contains
+- starts with
+- ends with
+- exists
+- not exists
+
+### Ignore failures [streams-ignore-failures]
+
+Turn on **Ignore failure** to ignore the processor if it fails. This is useful if you want to continue processing the document even if the processor fails.
+
+### Ignore missing fields [streams-ignore-missing-fields]
+
+Turn on **Ignore missing fields** to ignore the processor if the field is not present. This is useful if you want to continue processing the document even if the field is not present.
+
+### Preview changes [streams-preview-changes]
```

Comment:
Could it make sense to move this section up on the page? I guess I'm asking if previewing changes is part of a real-time feedback loop? Should users be aware of this as they configure conditions/fields so that they can see what happens as they configure instead of after they've added multiple configs?